lt_lakes_req changes

KarHarker opened the pull request:
modified lt_lakes_sites to make the default option query active not all sites, moved 2 sites (E275784, 1132490) from inactive to active list 

#59

KarHarker committed:
modified lt_lakes_sites to make the default option query active not all sites, moved 2 sites (E275784, 1132490) from inactive to active list

ateucher reviewed (commented):
Looks great! I think we can keep working through the checklist and merge it all as one once done.

KarHarker committed:
adding req id as a column to retain in wq_cols

KarHarker committed:
adding req_id as an option in filter_ems_data() function

KarHarker committed:
add lt_lake_req function to namespace

KarHarker committed:
updated filter_ems_data function to include req_id

KarHarker committed:
add lt_lake_req function to filter by req_id

KarHarker committed:
add id lookup table

KarHarker committed:
modify filter_ems_data function to include req_id

KarHarker committed:
switched order of ems_id selection options

ateucher commented on R/helpers.R:
```diff
@@ -47,22 +47,42 @@ lt_lake_sites <- function(which = c("all", "active", "inactive")) {
+#'
+#' @param data loads a site ID lookup table
+#'
+#' @return a character vector of req ids (EMS IDs)
```
It says EMS IDs here
```suggestion
#' @return a character vector of requisition ids 
```

KarHarker committed:
Update R/helpers.R

Co-authored-by: Andy Teucher <andy.teucher@gov.bc.ca>

KarHarker committed:
documentation, files for long-term lake dataset

KarHarker committed:
modified lt_lake_req and lt_lake_sites functions and documentation

KarHarker committed:
updated documentation of helpers.R

Merge branch 'lt_lakes_req' of https://github.com/bcgov/rems into lt_lakes_req

# Conflicts:
#	R/helpers.R

KarHarker committed:
Merge branch 'master' into lt_lakes_req

KarHarker committed:
updates to lt_lake functions and test files

KarHarker committed:
updating documentation

KarHarker committed:
adding test file for helpers.R

KarHarker committed:
changing test files to include req_ids

KarHarker committed:
added information on lt lakes functions to readme

KarHarker committed:
adding test for helper function data type

ateucher commented on R/data_ops.R:
```diff
@@ -32,11 +32,12 @@ bind_ems_data <- function(...) {
 #' @param param_code a character vector of parameter codes
 #' @param from_date A `Date`, `POSIXct`, `POSIXlt`, or a `character` string in a standard unambiguous format (e.g., `"YYYY/MM/DD"`)
 #' @param to_date A `Date`, `POSIXct`, `POSIXlt`, or a `character` string in a standard unambiguous format (e.g., `"YYYY/MM/DD"`)
+#' @param req_id A character vector of requisition id(s) of interest
```
Could add a reference to `lt_lake_req()` - If you put it in square brackets roxygen will create a hyperlink to the man page for that function (e.g., "`See [lt_lakes_req()] for a list of long-term lakes monitoring sites requisition IDs`")

KarHarker committed:
editing tests to include req_id, referencing lt_lake_req() function

KarHarker commented:
ok @ateucher - I think this is complete!

ateucher commented on R/data_ops.R:
```diff
@@ -32,11 +32,12 @@ bind_ems_data <- function(...) {
 #' @param param_code a character vector of parameter codes
 #' @param from_date A `Date`, `POSIXct`, `POSIXlt`, or a `character` string in a standard unambiguous format (e.g., `"YYYY/MM/DD"`)
 #' @param to_date A `Date`, `POSIXct`, `POSIXlt`, or a `character` string in a standard unambiguous format (e.g., `"YYYY/MM/DD"`)
+#' @param req_id A character vector of requisition id(s) of interest. See [lt_lake_req()] for a list on long-term lakes monitoring sites REQUISITION IDs
```
```suggestion
#' @param req_id A character vector of requisition id(s) of interest. See [lt_lake_req()] for a list of long-term lakes monitoring sites REQUISITION IDs
```

ateucher committed:
typo

ateucher commented:
Looks great @KarHarker! I fixed one small typo - so if you can pull and run `document()` and push it back up, once the checks all pass go ahead and merge!

KarHarker committed:
fix typo
